packages/react/src/components/Notification/__tests__/Notification-test.js

```diff
@@ -363,6 +363,87 @@ describe('StaticNotification', () => {
 });
 
 describe('Callout', () => {
+  // children?: ReactNode;
+  // lowContrast?: boolean;
+  // statusIconDescription?: string;
```

Comment:
are these still needed?

Also, the previous test on line 346 has the following:
```
// TODO: Remove StaticNotification tests when Callout moves to stable OR in
// v12, whichever is first. Ensure test parity on Callout.
```
Seems like we can remove now?

Reply:
Thanks! I removed those and updated the comment. I think it's a good idea to leave this single StaticNotification test in since it's still exported, just deprecated. I updated the comment to reflect removal in v12